Submitted By: Matthew Burgess (matthew at linuxfromscratch dot org)
Date: 2003-08-23
Initial Package Version: 2.5.31
Origin: James E. Jurach Jr.
        (Bug 769192 at http://sourceforge.net/projects/lex)
Description: Enables modutils-2.4.25 to compile without generating
             the infamous error message as detailed in the FAQ and
             archives.

--- flex-2.5.31.orig/flex.skl	2003-03-31 19:51:38.000000000 -0600
+++ flex-2.5.31/flex.skl	2003-07-10 11:01:40.000000000 -0500
@@ -3280,7 +3283,6 @@
 #undef yy_set_bol
 #undef yy_new_buffer
 #undef yy_set_interactive
-#undef yytext_ptr
 #undef YY_DO_BEFORE_ACTION
 
 #ifdef YY_DECL_IS_OURS
